README.md

```diff
@@ -24,25 +17,89 @@ Change directory to the package under development. Note: an integration is a spe
 
 ```bash
 cd integrations
-cd package/my-integration
+cd package/my-package
 ```
 
-Run the `help` command and see available actions:
+Run the `help` command and see available commands:
 
 ```bash
 elastic-package help
 ```
 
-## GitHub authorization
+## Commands
```

Comment:
Do you think we can sort these commands and maybe keep help at the top of them?

Reply:
Sorted in 51c497473dffecb4d6163351d23d6b2e5bcb5e25.